ja-jp/Content/index.htm

```diff
@@ -284,7 +284,7 @@ <h1 style="margin-top:35px;">Oktaのドキュメント</h1>
                                                     </div>
                                                         <div class="tiles">
                                                             <div>
-                                                                <p class="tile-title" data-mc-conditions="Primary.live-site-only"><a href="../../oag/en-us/Content/Topics/Access-Gateway/ag-main.htm"><span class="mc-variable okta-feature-names.Access_Gateway variable">Access Gateway（英語のみ）</span></a>
+                                                                <p class="tile-title" data-mc-conditions="Primary.live-site-only"><a href="../../oag/ja-jp/Content/Topics/Access-Gateway/ag-main.htm"><span class="mc-variable okta-feature-names.Access_Gateway variable">Access Gateway</span></a>
```

Comment:
I see you are fixing the link, but should the text remain in Japanese? `Access Gateway（英語のみ）`

Reply:
AFAIK, the product names remain in English (DNT). So, here, I just removed the "(English only)" JA text.

Reply:
the official product name, which is in our [DNT](https://docs.google.com/spreadsheets/d/1vSpIrL0htS3cwFprHF7imGDQfLqxZNCimsfpumVftCY/edit#gid=1344639332), is "Okta Access Gateway", so that explains why they send translations for just "Access Gateway".
Can we update our product name to "Okta Access Gateway"? Otherwise let's chat with Yasmin next week to see if we can avoid them translating "Access Gateway"

Reply:
We should fix the H.O.C. EN landing page?
https://deploy-preview-1024--extraordinary-horse-71260e.netlify.app/en-us/content/

Reply:
Thanks for the translation terminology list. I'm going to update the EN H.O.C. landing page accordingly.